### ISSUE TITLE
make the games shorter

### ISSUE THREAD

**Issue by stephenfournier** [stephenfournier/dotastrikers#159]
there is no reason for games to be more than 25 mins. i'd prefer games to be on average around 20 mins. there are likely a variety of factors that are making games long that need to be looked into.

**Comment by CarpeSwag** [comment]
Yeah. I was actually going to start working on some of these changes. 

Off the top of my head here's a few things I was going to start changing to make the games go a little faster:

* Ball collection distance down a bit. This will hurt goalkeepers a tad and additionally removes the awful ass-grab steals where you just steal the ball through them. Like, both of you are facing the same direction and you approach from behind and somehow steal the ball from behind them.
* Referee issues. If you can't ref juggle as hard it'll decrease the games a bit
* Ball physics - Decreasing friction (but increasing the threshold for the ball to stop) so it goes farther, at a faster speed overall. Ball speed itself won't be changed, however, because old ball speed was too fast
